var/spack/repos/builtin/packages/py-openpmd-viewer/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyOpenpmdViewer(PythonPackage):
+    """Python visualization tools for openPMD files"""
+
+    homepage = "https://www.openPMD.org"
+    git      = "https://github.com/openPMD/openPMD-viewer.git"
+    pypi     = 'openPMD-viewer/openPMD-viewer-1.2.0.tar.gz'
+
+    maintainers = ['RemiLehe', 'ax3l']
+
+    version('1.2.0', sha256='a27f8ac522c4c76fd774095e156a8b280c9211128f50aa07f16ac70d8222384d')
+
+    variant('backend', default='h5py,openpmd-api', multi=True,
+            values=('h5py', 'openpmd-api'))
+    variant('jupyter', default=False,
+            description='Enable Jupyter Widget GUI')
+    variant('numba', default=False,
+            description='Enable accelerated depositions for histograms')
+    variant('plot', default=True,
+            description='Enable plotting support')
+    variant('tutorials', default=True,
+            description='Enable dependencies for tutorials')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@3.5:', type=('build', 'run'))
```
According to the `setup.py`